sdk/tests/DeserializationTests.cc

```diff
@@ -0,0 +1,254 @@
+{
+  // Given
+  const auto testAccountIdTo = getTestAccountIdTo();
+  const auto testAccountIdFrom = getTestAccountIdFrom();
+  const auto testTransactionIdFrom = TransactionId::generate(testAccountIdFrom).toProtobuf().release();
+  const int64_t testTransferAmount = 10LL;
+  const std::string testTxHash = "txHash";
+  const std::chrono::system_clock::time_point now = std::chrono::system_clock::now();
+  const std::string testTxMemo = "txMemo";
+  const uint64_t testTxFee = 10ULL;
+  proto::TransactionRecord testProtoTransactionRecord;
+  testProtoTransactionRecord.mutable_receipt()->set_allocated_accountid(testAccountIdFrom.toProtobuf().release());
+  testProtoTransactionRecord.set_allocated_transactionhash(new std::string(testTxHash));
+  testProtoTransactionRecord.set_allocated_consensustimestamp(internal::TimestampConverter::toProtobuf(now));
+  testProtoTransactionRecord.set_allocated_transactionid(testTransactionIdFrom);
+  testProtoTransactionRecord.set_allocated_memo(new std::string(testTxMemo));
+  testProtoTransactionRecord.set_transactionfee(testTxFee);
+  
+  proto::AccountAmount *testProtoAccountAmount = testProtoTransactionRecord.mutable_transferlist()->add_accountamounts();
+  testProtoAccountAmount->set_allocated_accountid(testAccountIdFrom.toProtobuf().release());
+  testProtoAccountAmount->set_amount(-testTransferAmount);
+  
+  testProtoAccountAmount = testProtoTransactionRecord.mutable_transferlist()->add_accountamounts();
+  testProtoAccountAmount->set_allocated_accountid(testAccountIdTo.toProtobuf().release());
+  testProtoAccountAmount->set_amount(testTransferAmount);
+
+  // When
+  TransactionRecord txRecord = TransactionRecord::fromProtobuf(testProtoTransactionRecord);
+  
+  // Then
+  EXPECT_TRUE(txRecord.getReceipt());
+  EXPECT_TRUE(txRecord.getReceipt()->getAccountId());
+  EXPECT_EQ(*txRecord.getReceipt()->getAccountId(), testAccountIdFrom);
+  EXPECT_EQ(txRecord.getTransactionHash(), testTxHash);
+  EXPECT_TRUE(txRecord.getConsensusTimestamp());
+  EXPECT_EQ(txRecord.getConsensusTimestamp()->time_since_epoch().count(), now.time_since_epoch().count());
+  EXPECT_TRUE(txRecord.getTransactionId());
+  EXPECT_EQ(txRecord.getTransactionId()->getAccountId(), testAccountIdFrom);
+  // EXPECT_GE(txRecord.getTransactionId().value().getValidTransactionTime(), now);
+  EXPECT_EQ(txRecord.getTransactionMemo(), testTxMemo);
+  EXPECT_EQ(txRecord.getTransactionFee(), testTxFee);
+  EXPECT_FALSE(txRecord.getTransferList().empty());
+  EXPECT_EQ(txRecord.getTransferList().at(0).getAccountId(), testAccountIdFrom);
+  EXPECT_EQ(txRecord.getTransferList().at(0).getAmount().toTinybars(), -testTransferAmount);
+  EXPECT_EQ(txRecord.getTransferList().at(1).getAccountId(), testAccountIdTo);
+  EXPECT_EQ(txRecord.getTransferList().at(1).getAmount().toTinybars(), testTransferAmount);
+}
+
+TEST_F(DeserializationTests, DeserializeTransactionResponseFromProtobufTest)
```

Comment:
To write this test I used test [ProtobufTransactionResponse](https://github.com/hashgraph/hedera-sdk-cpp/blob/c54139e8bc8f507e1f08d052b15eec3d188a0bd0/sdk/tests/TransactionResponseTest.cc#L31) from [TransactionResponseTest.cc](https://github.com/hashgraph/hedera-sdk-cpp/blob/main/sdk/tests/TransactionResponseTest.cc).

Reply:
Remove test `DeserializeTransactionResponseFromProtobufTest` from `DeserializationTests.cc` in commit https://github.com/hashgraph/hedera-sdk-cpp/pull/176/commits/24e9524b0d8a82ce38fd17d6a04f8255a3d5e480.